roles/test_operator/README.md

```diff
@@ -0,0 +1,45 @@
+# Test operator
```

Comment:
forgot to ask... so is the plan that we wire this up in those jobs that are currently running tempest? i.e. they should start using this role instead?

Reply:
Yes, the goal would be to execute all the tests via the test-operator. We plan to have a demo for the test-operator at the end of the month.